frontend/src/entities/jobs/algorithms.tsx

```diff
@@ -156,11 +189,27 @@ const linearIntProperties = {
     scalingType: 'Linear',
 };
 
+const staticParameterProperties = {
+    type: HyperparameterType.STATIC,
+    typeOptions: [HyperparameterType.STATIC],
+};
+
+const integerParameterProperties = {
+    type: HyperparameterType.INTEGER,
+    typeOptions: [HyperparameterType.INTEGER, HyperparameterType.STATIC],
+};
+
+const categoricalParameterProperties = {
+    type: HyperparameterType.CATEGORICAL,
+    typeOptions: [HyperparameterType.CATEGORICAL, HyperparameterType.STATIC],
+};
+
 const continuousParameterProperties = {
     type: HyperparameterType.CONTINUOUS,
-    typeOptions: [HyperparameterType.STATIC, HyperparameterType.CONTINUOUS],
+    typeOptions: [HyperparameterType.CONTINUOUS, HyperparameterType.STATIC],
```

Comment:
These were previously ordered this way on purpose @dustins may remember the exact reasoning but I'm fairly certain it was related to differences between rendering the inputs on a training job form vs an hpo training job definition form. Just want to call this out so we can confirm this doesn't break anything.

Reply:
Understood. I will check that. This is the ordering in the console which is why I switched it, but it could have adverse affects on our training rendering, so I will check.